.gitignore

```diff
@@ -10,7 +10,6 @@
 build
 sklearn/datasets/__config__.py
 sklearn/**/*.html
-
```

Comment:
This blank line should be reinserted.

Reply:
As a piece of additional information, we don't want to bring a difference in the files which are not linked with the PR itself :)

Reply:
I will do the corrections and once it fine apply it to the other occurences in the module I have chosen